names/data/names_data.hcl

```diff
@@ -2477,13 +2339,11 @@ service "costoptimizationhub" {
     human_friendly      = "Cost Optimization Hub"
   }
 
-  client {
-    skip_client_generate = true
-  }
-
   endpoint_info {
-    endpoint_api_call        = "GetPreferences"
-    endpoint_region_override = "us-east-1"
+    endpoint_api_call = "GetPreferences"
+    endpoint_region_overrides = {
+      "aws" = "us-east-1"
+    }
```

Comment:
Is scoping these override to the commercial partition potentially a breaking change for other partitions? Perhaps we include a changelog entry noting the change?

Reply:
The manually written API client factories always did a partition check for `aws` (and others for Route 53) -- this map captures that.